.github/workflows/docstring_validation.yml

```diff
@@ -0,0 +1,50 @@
+name: Test Docstrings Validation
+
+on:
+  pull_request:
+    paths:
+      - "integration-tests/**"
+
+jobs:
+  betelgeuse:
+    name: "betelgeuse dry-run"
+    runs-on: ubuntu-latest
+    container:
+      image: fedora:latest
+
+    - uses: actions/checkout@v4
```

Comment:
this is out of the `steps`, and makes this yaml file invalid:
```
Invalid workflow file: .github/workflows/docstring_validation.yml#L9
You have an error in your yaml syntax on line 9
```
(see also https://github.com/RedHatInsights/insights-client/actions/runs/10600325105)